exampleCourse/questions/template/number-input/random-prompt/server.py

```diff
@@ -0,0 +1,28 @@
+import math
+import random
+
+
+def generate(data):
+    # Sample an integer number between 2 and 20.
+    a = random.randint(2, 20)
+    data["params"]["a"] = a
+
+    # Sample an integer number between 6 and 11.
+    b = random.randint(6, 11)
+    data["params"]["b"] = b
+
+    # Define a list of equations and their solutions.
+    equation_options = [
+        ("\\sqrt{(a^2 + b^2)}", math.sqrt(a**2 + b**2)),
+        ("a^2/b", a**2 / b),
+        ("2(b-a)^2", 2 * (b - a) ** 2),
+        ("(a^2 + b^2)/a", (a**2 + b**2) / a),
+        ("(a^2 + b^2)/b", (a**2 + b**2) / b),
```

Comment:
Feel free to ignore it, but since you're using Latex:

```suggestion
        (r"\sqrt{(a^2 + b^2)}", math.sqrt(a**2 + b**2)),
        (r"\cfrac{a^2}{b}", a**2 / b),
        ("2(b-a)^2", 2 * (b - a) ** 2),
        (r"\cfrac{a^2 + b^2}{a}", (a**2 + b**2) / a),
        (r"\cfrac{a^2 + b^2}{b}", (a**2 + b**2) / b),
```

Reply:
I like the usage of `r` strings to avoid escaping backslashes. I'll skip the `cfrac` changes for now.

Reply:
On second thought: I think `r` strings are just complex enough that it may not be worth putting in an example question? I'll skip this suggestion for now. We can discuss in a followup PR if you'd like to make this case for this.